website/data/docs-navigation.js

```diff
@@ -230,7 +230,7 @@ export default [
       {
         category: 'operations',
         name: 'Operations',
-        content: ['upgrading', 'tls-on-existing-cluster', 'uninstalling'],
+        content: ['upgrading', 'tls-on-existing-cluster', 'connect-ca-provider', 'uninstalling'],
```

Comment:
I think this makes more sense under the Connect sidebar because it's about configuring connect. At least that's where I'd look for this info.

Reply:
This was tough for me to decide, and I ended up choosing this section for a few reasons:
1) The connect's overview page already has the Installation and Configuration section, but it mostly refers to configuring the injector, rather than connect more generally. For example, there are no references there that connect must be enabled on the servers too.
2) There is no other obvious place where to put this under connect sidebar without a some refactor because it'd also be confusing to users if installation and configuration and vault ca are in different places. 

I think if we wanted to put it under connect, we'd need to refactor that whole section a bit because it'd be confusing if this page was separate from the rest of the installation instructions. 

What do you think?

Reply:
Yeah it's tough. Operations is weird for me because I think of that as day 2 tasks but this can only be done on initial installation.

I agree that it looks like we'd need to refactor the connect sidebar. I still think it would be better placed there even if it's weird and then we refactor the sidebar separately. But I'm about 60/40 on it so whatever you like is good too.

Reply:
Yeah, I definitely agree with you about Operations, and that it's also weird here. I'll place it under Connect for now. Even though it's a bit confusing, at least it's a more natural place to look for it.

Reply:
Moved!